manifest.webapp

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Files",
+  "name": "Cozy Drive",
```

Comment:
I would do s/Files/Drive/ as this manifest is used by the cozy-stack (in which the word Cozy is obvious)